server.go

```diff
@@ -22,11 +21,11 @@ var homeTemplate = template.Must(template.ParseFiles("./public/home.html"))
 var rnd *renderer.Render
 
 type wsConfig struct {
-	WsHost string
+	WsHost        string
+	StampedTweets []displayTweet
```

Comment:
We don't need to store this here. We already have access to `timestampedTweets` since this file is also part of the main package. We can just refer to that whenever we need it.